package.json

```diff
@@ -27,6 +24,7 @@
     "lucide-react": "^0.395.0",
     "luxon": "^3.4.4",
     "next": "14.0.3",
+    "ngeohash": "^0.6.3",
```

Comment:
Last updated 6 years ago 😬 If it works, it works though! Seems to be getting quite a lot of weekly downloads still. https://www.npmjs.com/package/ngeohash